src/main/java/org/folio/support/ExpirationTool.java

```diff
@@ -42,44 +46,67 @@ private ExpirationTool() {
     return future.compose(rs -> CompositeFuture.all(rs.getRows()
       .stream()
       .map(row -> doRequestExpirationForTenant(vertx, getTenant(row.getString("nspname"))))
-      .collect(Collectors.toList())));
+      .collect(Collectors.toList()))
+      .map(all -> null));
```

Comment:
Can you help me understand this change? What collect the set of future for each tenant and then map them to null?

Reply:
I made this change to make method return type more obvious, `Future<Void>` instead of `Future<CompostiteFuture>`. It means that we expect a future without any result(`Void` result). As `CompositeFuture.all` is used here the method result will succeed if all composed results are succeeded.

Reply:
Ok, thanks for explaining it to me